main.tf

```
@@ -76,6 +76,9 @@ resource "aws_db_instance" "database" {
   final_snapshot_identifier = "${var.identifier}-final-snapshot"
   copy_tags_to_snapshot     = true
 
+  performance_insights_enabled          = var.performance_insights != null ? true : false # If the object exists then turn on
```

(nit) can you just use the check as the boolean?
```suggestion
  performance_insights_enabled          = var.performance_insights != null # If the object exists then turn on
```